build.sbt

```diff
@@ -56,20 +57,23 @@ lazy val common = project.in(file("faunadb-common"))
     javacOptions ++= Seq("-source", "1.7", "-target", "1.7"),
     apiURL := Some(url(s"http://faunadb.github.io/faunadb-jvm/$driverVersion/faunadb-common/api/")),
 
-    (javacOptions in doc) := Seq("-source", "1.7", "-Xdoclint:none",
```

Comment:
The warnings were happening on the `java` module previously. Now, they should be happening on the `java-dsl` module. Can you generate its docs to confirm?